website/views/shared/profiles/achievements.jade

```diff
@@ -18,9 +18,9 @@ div(ng-if='::profile.achievements.habitSurveys || user._id == profile._id')
   .achievement.achievement-tree(ng-if='::profile.achievements.habitSurveys')
   div(ng-class='::{muted: !profile.achievements.habitSurveys}')
     h5=env.t('helped')
-    small(ng-if='profile.achievements.habitSurveys > 1')
-      =env.t('surveysMultiple', {surveys: "{{profile.achievements.habitSurveys}}"})
-    small(ng-if='!(profile.achievements.habitSurveys > 1)')
+    small(ng-if='::profile.achievements.habitSurveys > 1')
+      =env.t('surveysMultiple', {surveys: "{{::profile.achievements.habitSurveys}}"})
+    small(ng-if='!(::profile.achievements.habitSurveys > 1)')
```

Comment:
This one should be at the beginning, like this: `ng-if='::!profile.achievements.habitSurveys > 1'